
string name
bool detectFace
int32 deltaX
int32 deltaY
bool canRecognize
string emotion
int32 faceArea
